src/main/capi/profiling_info-c.cpp

```diff
@@ -23,6 +23,8 @@ duckdb_profiling_info duckdb_get_profiling_info(duckdb_connection connection) {
 	return reinterpret_cast<duckdb_profiling_info>(profiling_info_ptr);
 }
 
+// FIXME: this should return a duckdb_value, so we can properly display structured data
+// at the moment everything has to be flattened to a string
```

Comment:
Since the `duckdb_profiling_info_get_value` is newly introduced, we should do this before we release a new version as we can still change the API of the function. Could you perhaps create an internal issue to track this with as milestone `1.1`?